### ISSUE TITLE
Accesibility testing and findings of VoiceOver and Larger Text

### ISSUE THREAD

**Issue by devinreams** [mozilla-lockbox/lockbox-ios#441]
tldr: everything is fairly good (thanks, Apple!) and just a few areas to consider improving:

- [x] the filter sort button on list view does not suggest what it is/does when in 'A-Z' mode relies on #453
- [x] the reveal password button on detail view does not suggest what its revealing relies on #453
- [x] the "learn more" usage data link does not appear accessible to open when VoiceOver relies on #453
- [x] the on screen alerts (password copied, syncing items) are not read on screen / accessible relies on #454
- [x] the "search your entries" is not tappable at the magnifying glass icon and all the way to right edge #441
- [x] most buttons and text labels do not respect and adjust to "larger text" / dynamic sizes #442

## Welcome Screen (first run)

- "Firefox Lockbox .. take your passwords everywhere"  is read
- "Get Started .. button"  is read
- "To use Firefox Lockbox..."  is read
- "Learn more ... link .. image"  is read

## FxA web view

- "close button"  is read
- "Sign in.. to continue to Firefox Sync" (on screen content)  is read
- "secure text field.. password... double tap to edit" (form fields) are read
- "show password .. checkbox" is read
  - ⚠️ "show password" eye icon is not seemingly able to toggle with double tap during VoiceOver since it's a "tap to hold" interaction? (@rbillings)

## Welcome Screen (already signed in)

- "Firefox Lockbox ... take your passwords everywhere" is read
- "alert .. Face ID authenticator" is read
   - "Face ID authenticated" (success) is read

## Entry List

- "No entries found" blurb is read
- ⚠️ "No entries found" text size does not adjust to larger text sizes
- ⚠️ "A-Z" sort button does not hint it is for sort/ordering and "A Z button is read"
- ⚠️ "Syncing entries" alert is not read
- "Firefox Lockbox" header is read 
- "Preferences button" is read
- "Search your entries text field... tap to edit" is read
  - ⚠️ magnifying glass tap does not go into text field (left side tap not accessible)
  - ⚠️ right side placeholder for "Cancel" button means text field does not extend (right side tap less accessible)
- ⚠️ placeholder text and search field does not adjust to larger text sizes
- ⚠️ table cells do not adjust to larger text sizes

## Item Detail

- "back button" is read
- "... heading" is read
- "Web address.." is read
- "username.." is read
- "password.. 15 bullets" is read
- "reveal eye button" is read
  - ⚠️ reveal eye button does not suggest what it is used to reveal
- ⚠️ username/password copied alert is not read
- "learn how to edit this entry..." is read
  - not noted as a link or image yet, but in progress at #430 
- ⚠️ table cells do not adjust to larger text sizes
- ⚠️ alert text does not adjust to larger text sizes

## Settings

- "..heading" is read
- "support heading" is read
- "send feedback button" is read
  - "settings back button" is read
- "frequently asked questions button" is read
  - "settings back button" is read
- "send usage data..." is read
  - "switch button.. on .. double tap to toggle setting" is read
  - ⚠️ double tap toggles setting, but "learn more" link seems impossible to access
- "account button" is read
- "auto lock ... on app exit ... button" is read
- "open websites in ... Firefox ... button" is read
- "lock now .. button" is read
  - ⚠️ "lock now button" does not follow large accessibility text size adjustment
- table cells adjust to larger text sizes

## Account Settings

- ⚠️ no text adjusts to larger text sizes
- "settings button" (back) is read
- "to make changes to your account..." is read
- "disconnect ... button" is read
- "disconnect firefox lockbox?" is read
  - "cancel button" is read
  - "disconnect button" is read

## Auto Lock Settings

- ⚠️ "Sign out of Firefox Lockbox" does not adjust to larger text sizes
- "settings button" (back) is read
- "sign out of .. after"  is read
- "on app exit .. selected"  is read
- "1 minute"...  is read

## Browser Settings

- "settings button" (back) is read
- "selected .. Firefox" is read
- "Firefox Focus"... is read

---

FYI: @rbillings (FxA PI) and @csuciu @SimonBasca @isabelrios (PI)

**Comment by devinreams** [comment]
Findings addressed and follow-up issues filed
